.travis.yml

```diff
@@ -14,9 +14,12 @@ notifications:
 
 install:
   - go get -u github.com/Masterminds/glide
+  - rm -rf vendor/
```

Comment:
We grab source, we remove vendor dir, we refetch dependencies according to lock file, we check if vendor dir is equal to fetched dependencies.